src/Chunk.ts

```diff
@@ -245,26 +245,7 @@ export default class Chunk {
 		this.orderedModules.forEach(module => {
 			Object.keys(module.imports).forEach(importName => {
 				const declaration = module.imports[importName];
-
-				const tracedExport = this.traceExport(declaration.module, declaration.name);
-
-				// ignore imports to modules already in this chunk
-				if (!tracedExport || tracedExport.module.chunk === this) {
-					return;
-				}
-
-				const variable = tracedExport.module.traceExport(tracedExport.name);
-
-				// namespace variable can indicate multiple imports
-				if (tracedExport.name === '*') {
-					Object.keys((<NamespaceVariable>variable).originals || (<ExternalVariable>variable).module.declarations).forEach(importName => {
-						const original = ((<NamespaceVariable>variable).originals || (<ExternalVariable>variable).module.declarations)[importName];
-						this.populateImport(original, tracedExport);
-					});
-					return;
-				}
-
-				this.populateImport(variable, tracedExport);
+				this.traceImport(declaration.module, declaration.name);
```

Comment:
Ah, I totally overlooked this one. So extraction actually removed a code duplication 👍

Reply:
Yup!